types/primer__octicons/index.d.ts

```diff
@@ -203,6 +206,7 @@ declare namespace octicons {
         | 'file-symlink-file'
         | 'file-zip'
         | 'filter'
+		| 'fiscal-host'
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/primer__octicons/index.d.ts;
  Line: 209;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.